docs/self-hosted-appcircle/install-server/helm-chart/configuration/advanced-configuration.md

```diff
@@ -125,523 +125,157 @@ webhook:
+| `global.mail.smtp.host`                                       | SMTP server hostname.                                                     | -                             |
+| `global.mail.smtp.username`                                   | SMTP username.                                                            | -                             |
+| `global.mail.smtp.from`                                       | The "From" address used for emails.                                       | -                             |
+| `global.mail.smtp.fromDisplayName`                            | The display name for the sender of emails.                                | -                             |
+| `global.mail.smtp.port`                                       | Port number for the SMTP server.                                          | -                             |
+| `global.mail.smtp.password`                                   | Password for the SMTP account.                                            | -                             |
+| `global.mail.smtp.ssl`                                        | Whether SSL is enabled for SMTP.                                          | 'false'                       |
+| `global.mail.smtp.auth`                                       | Whether Authentication is enabled for SMTP.                               | 'true'                        |
+| `global.mail.smtp.starttls`                                   | Whether STARTTLS is enabled for SMTP.                                     | 'true'                        |
+| `global.distribution.distribution-testerweb.url`              | The external URL for the distribution tester web module.                  | -                             |
+| `global.tlsWildcard.cert`                                     | The wildcard TLS certificate.                                             | -                             |
+| `global.tlsWildcard.caCert`                                   | The Certificate Authority (CA) for the wildcard certificate.              | -                             |
+| `global.tlsWildcard.key`                                      | The private key for the wildcard certificate.                             | -                             |
+| `global.trustedCerts`                                         | List of trusted certificates.                                             | []                            |
+| `global.minio.url`                                            | External Minio URL.                                                       | -                             |
+| `global.minio.region`                                         | The region for Minio.                                                     | `local`                       |
+| `global.minio.bucketPrefix`                                   | Prefix for Minio buckets.                                                 | `appcircle-local-resource-`   |
```

Comment:
- [ ] Special names, terminology and concepts should be compatible with the rest of documentation. (For instance, use "MinIO" instead of "Minio")

Please review the whole table (descriptions) with this perspective.